.changeset/metal-avocados-juggle.md

```diff
@@ -0,0 +1,28 @@
+---
+'@keystonejs/adapter-knex': patch
+'@keystonejs/adapter-mongoose': patch
+'@keystonejs/apollo-helpers': patch
+'@keystonejs/app-admin-ui': patch
+'@keystonejs/app-graphql': patch
+'@keystonejs/app-next': patch
+'@keystonejs/app-nuxt': patch
+'@keystonejs/app-static': patch
+'@keystonejs/app-version': patch
+'@keystonejs/auth-passport': patch
+'@keystonejs/auth-password': patch
+'create-keystone-app': patch
+'@keystonejs/email': patch
+'@keystonejs/fields-auto-increment': patch
+'@keystonejs/fields-markdown': patch
+'@keystonejs/fields-mongoid': patch
+'@keystonejs/fields': patch
+'@keystonejs/file-adapters': patch
+'@keystonejs/keystone': patch
+'@keystonejs/list-plugins': patch
+'@keystonejs/oembed-adapters': patch
+'@keystonejs/session': patch
+'@keystonejs/test-utils': patch
+'@keystonejs/website': patch
```

Comment:
Hmm, should this only `patch` the website?

Reply:
Yep

Reply:
(and nothing else)